GRAPHICS/DisableMoonCycleOverride.md

```diff
@@ -0,0 +1,23 @@
+---
+ns: GRAPHICS
+aliases: ["0x2BF72AD5B41AA739", "_RESET_EXTRA_TIMECYCLE_MODIFIER_STRENGTH"]
+---
+
+## DISABLE_MOON_CYCLE_OVERRIDE
+
+```c
+// 0x2BF72AD5B41AA739 b323
```

Comment:
You should remove the game build version (`b323`), that used to be reserved for joaat native hashes.

It can be seen on the documentation for these two natives.

Reply:
done.